tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal

```diff
@@ -246,6 +246,37 @@ function testSpreadFieldWithRecordTypeHavingNeverField() {
     assertEquality(8, bar8.i);
 }
 
+type Vehicle record {
+    string model;
+    int year;
+};
+
+type Car record {|
+    string model;
+    int year;
+    anydata ...;
+|};
+
+type Van record {|
+    string model;
+    int year;
+    any ...;
+|};
+
+function testSpreadFieldWithClosedRecordCreatedFromOpenRecord() {
```

Comment:
Van is not a closed record.

https://ballerina.io/spec/lang/master/#section_5.4.2.2

> A record type descriptor that either is an inclusive-record-type-descriptor or is an exclusive-record-type-descriptor with a record-rest-descriptor is called open; a record type descriptor that is not open is called closed.